### PR TITLE
MOM: Jin-Gitaxias // The Great Synthesis

### PR DESCRIPTION
**Pull request by MikeS-NZ**
MOM: Add card script for new card Jin-Gitaxias // The Great Synthesis

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/jin_gitaxias_the_great_synthesis.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Name:The Great Synthesis
+ManaCost:no cost
+Colors:blue
+Types:Enchantment Saga
+K:Saga:3:DBDraw,DBBounceAll,DBCastAll
+SVar:DBDraw:DB$ Draw | NumCards$ Y | SubAbility$ DBEffect | SpellDescription$ Draw cards equal to the number of cards in your hand. You have no maximum hand size for as long as you control CARDNAME.
+SVar:DBEffect:DB$ Effect | StaticAbilities$ STHandSize | Duration$ UntilLoseControlOfHost | SpellDescription$ You have no maximum hand size.
+SVar:STHandSize:Mode$ Continuous | EffectZone$ Command | Affected$ You | SetMaxHandSize$ Unlimited | Description$ You have no maximum hand size.
+SVar:DBBounceAll:DB$ ChangeZoneAll | ChangeType$ Creature.nonPhyrexian | Origin$ Battlefield | Destination$ Hand | SpellDescription$ Return all non-Phyrexian creatures to their owners' hands.
+SVar:DBCastAll:DB$ Play | Valid$ Card.nonLand | ValidZone$ Hand | WithoutManaCost$ True | Amount$ SynthesisX | Controller$ You | Optional$ True | SubAbility$ DBExileSelf | SpellDescription$ You may cast any number of spells from your hand without paying their mana costs. Exile CARDNAME, then return it to the battlefield.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/jin_gitaxias_the_great_synthesis.txt:29]: `ValidSA$ Spell`

Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/jin_gitaxias_the_great_synthesis.txt:29]: Just use `Amount$ All`

Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/jin_gitaxias_the_great_synthesis.txt:29]: `+YouOwn`

File: forge-gui/res/cardsfolder/upcoming/jin_gitaxias_the_great_synthesis.txt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+SVar:DBExileSelf:DB$ ChangeZone | Origin$ Battlefield | Destination$ Exile | SubAbility$ DBReturnSelf | RememberChanged$ True | SpellDescription$ Exile CARDNAME, then return it to the battlefield under its owner's control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/jin_gitaxias_the_great_synthesis.txt:30]: No reason to repeat SpellDescription?

File: forge-gui/res/cardsfolder/upcoming/jin_gitaxias_the_great_synthesis.txt (hunk 1 above)
Comment (MikeS-NZ) [reply forge-gui/res/cardsfolder/upcoming/jin_gitaxias_the_great_synthesis.txt]: Added this tag, thank you!

Comment (MikeS-NZ) [reply forge-gui/res/cardsfolder/upcoming/jin_gitaxias_the_great_synthesis.txt]: Fixed!

Comment (MikeS-NZ) [reply forge-gui/res/cardsfolder/upcoming/jin_gitaxias_the_great_synthesis.txt]: Added `+YouOwn` descriptor

File: forge-gui/res/cardsfolder/upcoming/jin_gitaxias_the_great_synthesis.txt (hunk 2 above)
Comment (MikeS-NZ) [reply forge-gui/res/cardsfolder/upcoming/jin_gitaxias_the_great_synthesis.txt]: Caught, thanks!

**Review by tool4ever — APPROVED** [review]

**Pull request merged**